corehq/apps/app_manager/templates/app_manager/partials/report_configs_js.html

```diff
@@ -14,10 +14,12 @@
             saveURL: saveURL,
             lang: {{ lang|JSON }},
             menuImage: menuImage,
-            menuAudio: menuAudio
+            menuAudio: menuAudio,
+            containerId: "#settings"
         });
         ko.applyBindings(reportModule, document.getElementById("save-button"));
         ko.applyBindings(reportModule, document.getElementById("module-name"));
+        ko.applyBindings(reportModule, document.getElementById("module-filter"));
```

Comment:
This breaks the Report Module page when `MODULE_FILTER` flag is not set, because then `document.getElementById("module-filter")` evaluates to `null` and `ko.applyBindings(foo, null)` applies the bindings to the entire body (unfortunately), which results in areas governed by two different bindings, which knockout rightly doesn't allow.

I stumbled upon this when testing own a mobile ucr feature I'm writing. https://github.com/dimagi/commcare-hq/pull/9339 should fix it